src/candidateInfo.css

```diff
@@ -148,7 +148,7 @@
     }
 
     .sc-modal-close {
-        position: absolute;
+        position: fixed;
```

Comment:
I'm surprised that a position `fixed` would improve embedding 😆 what's the specific bug this resolved?